src/main/java/seedu/duke/template/LrTemplate.java

```diff
@@ -0,0 +1,27 @@
+package seedu.duke.template;
+
+import seedu.duke.component.Inductor;
+import seedu.duke.component.Resistor;
+import seedu.duke.component.VoltageSource;
+
+public class LrTemplate extends Template {
+    private Resistor resistor;
+    private Inductor inductor;
+    private static final String RL_TEMPLATE = "\t+---R-----L---+\n"
```

Comment:
Consider moving the `static` constant to before the instance variables?
https://se-education.org/guides/conventions/java/index.html#classes-and-interfaces

Reply:
Okay will do!